tools/ci.psm1

```diff
@@ -792,7 +792,7 @@ function Invoke-LinuxTests
             # Create and package Raspbian .tgz
             Start-PSBuild -PSModuleRestore -Clean -Runtime linux-arm -Configuration 'Release'
             $armPackage = Start-PSPackage @packageParams -Type tar-arm -SkipReleaseChecks
-	        Copy-Item $armPackage -Destination "${env:BUILD_ARTIFACTSTAGINGDIRECTORY}" -Force
+	    Copy-Item $armPackage -Destination "${env:BUILD_ARTIFACTSTAGINGDIRECTORY}" -Force
```

Comment:
Again. :-) 
(can miss this enchanted string? :-) )

Reply:
In vscode it is indented properly, I think it’s just github

Reply:
Perhaps a invisible control symbol. Please remove all start spaces and retype.

Reply:
Will do.

Reply:
Does it look fixed now @iSazonov ?  I tried deleting the line and reindenting it.

Reply:
Yes, now looks good!

Reply:
Great, thanks for the review :)